addons/io_hubs_addon/third_party/recast.py

```diff
@@ -0,0 +1,636 @@
+# ***** BEGIN GPL LICENSE BLOCK *****
+#
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.	See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+#
+# ***** END GPL LICENCE BLOCK *****
+
+import os
+import traceback
+import bpy
+
+from bpy.props import IntProperty, FloatProperty, EnumProperty, PointerProperty, FloatVectorProperty, BoolProperty
+from bpy.types import Panel, PropertyGroup
+from mathutils import Matrix, Vector
+from math import radians
+from ..preferences import get_addon_pref
+from ..components.utils import add_component, get_objects_with_component, has_component
+
+import ctypes
+import ctypes.util
+from ctypes import c_int, c_float
+
+import bmesh
+
+
+CELL_SIZE_DEFAULT = 0.166
+CELL_HEIGHT_DEFAULT = 0.10
+SLOPE_MAX_DEFAULT = radians(60)
+CLIMB_MAX_DEFAULT = 0.9
+AGENT_HEIGHT_DEFAULT = 1.70
+AGENT_RADIUS_DEFAULT = 0.5
+EDGE_MAX_LENGTH = 12.0
+EDGE_MAX_ERROR = 1.0
+REGION_MIN_SIZE = 1.0
```

Comment:
Turns out the defaults I was looking at weren't the right ones for this property and Spoke actually defaults to a minimum region size of 4.0 m2.  So sorry, my bad.